frontend/model/contracts/chatroom.js

```diff
@@ -562,12 +580,39 @@ sbp('chelonia/defineContract', {
         })
       }
     },
+    'gi.contracts/chatroom/deleteAttachment': {
+      validate: actionRequireInnerSignature(objectOf({
+        hash: string,
+        manifestCid: string
+      })),
+      process ({ data, innerSigningContractID }, { state }) {
+        if (!state.onlyRenderMessage) {
+          return
+        }
+        const msgIndex = findMessageIdx(data.hash, state.messages)
+        if (msgIndex >= 0 && innerSigningContractID === state.messages[msgIndex].from) {
```

Comment:
@corrideat correct me if I'm wrong but I believe this could/should also include a check for being the groupCreator?

Reply:
That's right, although in that case instead of a token the `billableContractID` attribute will be used

Reply:
We don't have any user flow for the group creators to delete attachments. So I would like to avoid this now.